pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -598,7 +602,7 @@ func (p *PriorityQueue) Pop() (*framework.QueuedPodInfo, error) {
 		// When Close() is called, the p.closed is set and the condition is broadcast,
 		// which causes this loop to continue and return from the Pop().
 		if p.closed {
-			return nil, fmt.Errorf(queueClosed)
+			return nil, queueClosedErr
```

Comment:
This error is not adding much value.

Maybe we can add a `log.Info` above and `return nil, nil`